packages/vuetify/src/components/VStepper/__tests__/VStepperContent.spec.ts

```diff
@@ -72,6 +77,28 @@ describe('VStepperContent.ts', () => {
     expect(wrapper.vm.computedTransition).toBe(VTabReverseTransition)
   })
 
+  it('should use opposite of reverse transition in rtl', () => {
+    const wrapper = mountFunction({
+      mocks: {
```

Comment:
you forgot to remove it here

Reply:
this to override and test it with RTL, should `$vuetify.rtl` be `true`

Reply:
ahh, sorry :)